compiler/src/main/resources/templates/generic_docker_preamble.ssp

```diff
@@ -19,17 +19,17 @@ if [[ ${bashDollar}num_lines != 1 ]]; then
     exit 1
 fi
 dx download ${bashDollar}{DOCKER_CREDENTIALS} -o ${bashDollar}{HOME}/docker_credentials
+
+# disable echo to avoid logging password
+set +e
```

Comment:
Shouldn't it be set +x / set -x for printing out commands, instead of +e/-e which exits upon errors.

Reply:
oops - you're right, good catch

Reply:
Np, sorry I missed it the first time.